### DISCUSSION TITLE
how to use NewStringNodeID

### DISCUSSION THREAD

**Discussion by nevernet** [gopcua/opcua#525] [Q&A] [answered]
https://github.com/gopcua/opcua/blob/299979655d394c9103283a686a4cebf4aca32471/ua/node_id.go#L56

what's exactly value should be filled in `id` parameter? 

```
// NewStringNodeID returns a new string node id.
func NewStringNodeID(ns uint16, id string) *NodeID {
	return &NodeID{
		mask: NodeIDTypeString,
		ns:   ns,
		bid:  []byte(id),
	}
}
```

**Comment by magiconair** [comment, marked as answer]
~~The namespace id.~~ The value of the string id, e.g. `foobar` to get a node id `s=foobar`.
